guides/common/modules/proc_republishing-repository-metadata-by-using-cli.adoc

```diff
@@ -0,0 +1,48 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="republishing-repository-metadata-by-using-cli"]
+= Republishing repository metadata by using CLI
+
+You can republish repository metadata when a repository distribution does not have the content that should be distributed based on the contents of the repository.
+
+Use this procedure with caution.
+{Team} recommends a complete repository sync or publishing a new content view version to repair broken metadata.
```

Comment:
Should we avoid referencing CV since we are in the `Republishing repository metadata by using CLI` section?
```suggestion
{Team} recommends a complete repository sync or metadata republish to repair broken metadata.
```

Reply:
I think this is exactly the point: You should not really (need to) do this: republish repo metadata. That's why this line mentions either doing a complete re-sync or publishing a new CV version.